crates/agglayer-node/src/rpc/debug.rs

```diff
@@ -0,0 +1,260 @@
+use std::sync::Arc;
+
+use agglayer_config::Config;
+use agglayer_storage::stores::{
+    DebugReader, DebugWriter, PendingCertificateReader, PendingCertificateWriter, StateReader,
+    StateWriter,
+};
+use agglayer_types::{
+    Certificate, CertificateHeader, CertificateId, CertificateStatus, Height, NetworkId,
+};
+use jsonrpsee::{
+    core::async_trait,
+    proc_macros::rpc,
+    server::{ServerBuilder, ServerHandle},
+};
+use tower_http::{compression::CompressionLayer, cors::CorsLayer};
+use tracing::{error, info, instrument};
+
+use super::error::RpcResult;
+use crate::rpc::{error::Error, rpc_middleware};
+
+#[rpc(server, namespace = "debug")]
+pub(crate) trait DebugAgglayer {
```

Comment:
nit: I am not sure `debug_` is quite fitting name given operations like removing certificates etc. Maybe something like `admin_` or `operator_` would be a better match?

Reply:
`admin` seems a better name yes !